internal/cli/atlas/customdbroles/create.go

```diff
@@ -72,7 +72,7 @@ func (opts *CreateOpts) validate() error {
 	return nil
 }
 
-// mongocli atlas dbrole(s) create <roleName> --privilege action[@dbName.collection] --inheritedRole role@db.
+// mongocli atlas dbrole(s) create <roleName> --privilege action@dbName.collection --inheritedRole role@db.
```

Comment:
the reason of `[]` is to denote it's optional

Reply:
ah okay, I thought It should be `[action@dbName.collection]` in that case but I see how the user can provide only `action`